modin/experimental/pandas/test/test_io_exp.py

```diff
@@ -89,10 +89,11 @@ def test_from_sql_defaults(make_sql_connection):  # noqa: F811
 
 @pytest.mark.usefixtures("TestReadGlobCSVFixture")
 @pytest.mark.skipif(
-    Engine.get() != "Ray", reason="Currently only support Ray engine for glob paths."
+    Engine.get() not in ("Ray", "Unidist", "Dask"),
+    reason=f"{Engine.get()} does not have experimental API",
```

Comment:
```suggestion
    reason=f"{Engine.get()} does not have experimental glob API",
```
we can have different experimental APIs, not all of which are enabled simultaneously